service/history/hsm/tree.go

```diff
@@ -244,8 +271,7 @@ func (n *Node) Outputs() []PathAndOutputs {
 func (n *Node) ClearTransactionState() {
 	root := n.root()
 	if root.opLog != nil {
-		root.opLog.DeletedPaths = nil
-		root.opLog.TransitionsByPath = make(map[string][]TransitionOutputWithCount)
+		root.opLog = make(OperationLog, 0)
```

Comment:
nit: this could be set to nil to avoid the allocation. len() and append() accept nils.

Reply:
Thx for the tip! Done